Alignment/OfflineValidation/test/test_all_cfg.py

```diff
@@ -174,17 +181,29 @@
 ####################################################################
 # Load and Configure TrackRefitter
 ####################################################################
-#import Alignment.CommonAlignment.tools.trackselectionRefitting as trackselRefit
-#process.seqTrackselRefit = trackselRefit.getSequence(process,'generalTracks')
-#process.seqTrackselRefit.TrackSelector.ptMin = cms.double(3)
+# process.load("RecoTracker.TrackProducer.TrackRefitters_cff")
+# import RecoTracker.TrackProducer.TrackRefitters_cff
+# process.FinalTrackRefitter = RecoTracker.TrackProducer.TrackRefitter_cfi.TrackRefitter.clone()
+# process.FinalTrackRefitter.src = "generalTracks"
+# process.FinalTrackRefitter.TrajectoryInEvent = True
+# process.FinalTrackRefitter.NavigationSchool = ''
+# process.FinalTrackRefitter.TTRHBuilder = "WithAngleAndTemplate"
```

Comment:
please remove the commented code

Reply:
As I am still not totally convinced that the CTSR sequence is OK to use in all corner use cases I prefer to keep track of the standard refitter configuration (at least in the test cfg: in the template of the all-ini-one it is removed)